builders/tools/explorers/overview.md

```diff
@@ -21,6 +21,7 @@ Moonbeam provides two different kind of explorers: ones to query the Ethereum AP
     |                                                        Block Explorer                                                        |   Type    |                           URL                           |
     |:----------------------------------------------------------------------------------------------------------------------------:|:---------:|:-------------------------------------------------------:|
     |                                   [Moonscan](https://moonbeam.moonscan.io/){target=_blank}                                   |    EVM    |              https://moonbeam.moonscan.io/              |
+    |                          [Blockscout](https://blockscout.moonbeam.network/){target=_blank}                          |    EVM    |      https://blockscout.moonbeam.network/      |
```

Comment:
can you please remove the commented stuff out for blockscout on lines 17 & 18

Reply:
Done. I'll do the checklist in the morning.